### PR TITLE
script to transfer response templates

### PR DESCRIPTION
**Pull request by struan**
Enables transferring of response templates between two installs.

### CODE REVIEW

**Review by davea — CHANGES_REQUESTED** [review]
Not sure if the same(ish) file being present twice is deliberate? Given the strong overlap with `export-import-data` I think this would be better implemented as a flag (e.g. `--update-templates`) on that existing script.

**Review by davea — CHANGES_REQUESTED** [review]
Sorry, I might be missing something but it looks like the flag isn't taken into account on L138 of `export-import-data` so existing templates are always skipped. Also `bin/transfer-updates-response-templates` is still hanging around.

**Review by davea — CHANGES_REQUESTED** [review]
This is crashing when trying to e.g. reimport a file I just exported:

```
DBIx::Class::Storage::DBI::_dbh_execute(): DBI Exception: DBD::Pg::st execute failed: ERROR:  duplicate key value violates unique constraint "response_templates_body_id_title_key"
DETAIL:  Key (body_id, title)=(21076, Action Scheduled - Blocked Drain) already exists.
[...]
 at ./bin/export-import-data line 148
```

File: bin/export-import-data [hunk 1]
```diff
@@ -130,11 +134,11 @@ sub import {
 
     foreach (@{$out->{templates}}) {
         my $existing = $body->response_templates->search({ title => $_->{title} })->single;
-        if ($existing) {
+        if ($existing && !$update_templates) {
             warn "Template with title $_->{title} already exists, skipping";
             next;
         }
-        my $template = $body->response_templates->new({
+        my $template = $body->response_templates->find_or_new({
```
Comment (davea) [inline bin/export-import-data:141]: This won't update the text or status of any existing templates unfortunately. `find_or_new` will (I believe) only search on unique columns, meaning it'll match on `body_id` and `title` then skip over the values for `text` and `state`.

**Review by davea — CHANGES_REQUESTED** [review]

File: bin/export-import-data (hunk 1 above)
Comment (struan) [reply bin/export-import-data]: sigh, yes :(

**Review by davea — APPROVED** [review]
🌟

**Pull request merged**